main.c

```diff
@@ -69,6 +69,22 @@ int main(void)
     MSS_TIM64_enable_irq();
     NVIC_SetPriority(Timer1_IRQn, 1);
 
+    /* load citi conf on startup */
+    uint32_t cfg_id_32 = 0;
+    mem_read(NVM_CITIROC_CONF_NO, &cfg_id_32);
+    uint8_t cfg_id = (uint8_t)(cfg_id_32);
```

Comment:
This should be a global variable, i.e., declared outside the function and redeclared as `extern` in every file it is in use (e.g., `msp_exp_handler.c`).